modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -145,7 +145,22 @@ Configuration example:
 ----
 | No
 | `target` | The CloudHub 2.0 target name to deploy the app to. +
-Specify either a shared space or a private space available in your Deployment Target values in CloudHub 2.0. See xref:cloudhub-2::ch2-features.adoc[Features of CloudHub 2.0] for a detailed description on shared and private spaces. | Yes
+Specify either a shared space or a private space available in your Deployment Target values in CloudHub 2.0. Below are target values for shared space:
+
+* `Cloudhub-US-East-1` (US East, N. Virginia)
```

Comment:
Are you sure here is the name instead of the id?
If the id, then for shared spaces it is equal to the name but all lowercases.
Private spaces id is an uuid.

Reply:
@juanmanuelmenendez - target field accept name instead of Id. As it is using name, values are defined in pull request